MainRepository/Program.cs

```diff
@@ -1,7 +1,44 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿// Implementation of Anagram
+class Anagram {
+
+   public static void Main (string[] args) {
+      Anagram a = new Anagram ();
+      a.test ();
+   }
+
+   // Function to find given two strings are anagram or not
+   bool IsAnagram (string str1, string str2) {
+      if (str1.Length != str2.Length) return false;

```

Comment:
This implementation is complex.
Could have simplified by simple equal check after sorting the characters in ascending order